v2/civ2.py

```diff
@@ -66,6 +66,8 @@ def main():
         if opts.down == True:
             ci.down()
         sys.exit(1)
+    finally:
+        ci.collectWindowsLogs()
```

Comment:
The problem here is that we are collecting logs no matter what happens with the cluster. Say we have an error somewhere along the way, either in up() or testing(). This will raise and exception that will trigger the cluster to be deleted ( L66-L67 ). Where are we collecting logs from then? 
 After we fix this, we still need some error checking in collectWindowsLogs(). What if deployment fails at up() and there are no accessible machines to collect logs from?